mappings/net/minecraft/util/math/Direction.mapping

```diff
@@ -58,7 +58,7 @@ CLASS ey net/minecraft/util/math/Facing
 		METHOD b isVertical ()Z
 		METHOD c isHorizontal ()Z
 		METHOD m asString ()Ljava/lang/String;
-	CLASS ey$b Direction
+	CLASS ey$b AxisDirection
```

Comment:
Any better name for this? I initially called this direction because I don't know if there's such a word that expresses if a scalar/number is positive or negative.

Reply:
In Polish, we have "kierunek" and "zwrot" for the two. English does not really have the concept - some people use "magnitude" to refer to the latter, but it's kind of artificial in a binary context.

Sign?

Reply:
> I don't know if there's such a word that expresses if a scalar/number is positive or negative

I would have said Polarity, but apparently that's a Chemistry word :S 

Anyhow should probably be `Direction.Axis` not `Direction.AxisDirection`.

Reply:
In Mathematics, the word for this would be "orientation".